ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -300,6 +301,15 @@ public boolean onTouchEvent(MotionEvent ev) {
     return super.onTouchEvent(ev);
   }
 
+  @Override
+  public boolean executeKeyEvent(KeyEvent event) {
+    int eventKeyCode = event.getKeyCode();
+    if (!mScrollEnabled && (eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN || eventKeyCode == KeyEvent.KEYCODE_DPAD_UP)) {
```

Comment:
And for consistency, can we check `KEYCODE_DPAD_UP` first? So that it's `KEYCODE_DPAD_LEFT` and `KEYCODE_DPAD_RIGHT`, `KEYCODE_DPAD_UP` and `KEYCODE_DPAD_DOWN`:

```
eventKeyCode == KeyEvent.KEYCODE_DPAD_UP ||
    eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN
```